src/service.js

```diff
@@ -10,10 +10,20 @@
     app.service('vcRecaptchaService', ['$window', '$q', function ($window, $q) {
         var deferred = $q.defer(), promise = deferred.promise, recaptcha;
 
-        $window.vcRecaptchaApiLoaded = function () {
+        if(angular.isUndefined($window.vcRecaptchaApiLoaded)){
+            $window.vcRecaptchaApiLoaded = []
+        }
+        
+        $window.vcRecaptchaApiLoaded.push(function () {
             recaptcha = $window.grecaptcha;
 
             deferred.resolve(recaptcha);
+        });
+
+        $window.vcRecaptchaApiLoadedCallback = function () {
+            $window.vcRecaptchaApiLoaded.forEach(function(element,index,array) {
```

Comment:
use var names that having meaning, `element` is not meaningful here. Perhaps `callback` or `cb`. There is no need for the other variables as they are not used. Yes, they will be removed in minification, but eh.